x/mint/test/mint_test.go

```diff
@@ -0,0 +1,148 @@
+package test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/celestia-app/test/util/testnode"
+	minttypes "github.com/celestiaorg/celestia-app/x/mint/types"
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/gogo/protobuf/proto"
+	"github.com/stretchr/testify/suite"
+	"github.com/tendermint/tendermint/rpc/client"
+)
+
+type IntegrationTestSuite struct {
+	suite.Suite
+
+	cctx testnode.Context
+}
+
+func (s *IntegrationTestSuite) SetupSuite() {
+	if testing.Short() {
+		s.T().Skip("skipping mint integration test in short mode.")
+	}
+
+	t := s.T()
+	t.Log("setting up mint integration test suite")
+
+	cparams := testnode.DefaultParams()
+	oneDay := time.Hour * 24
+	oneMonth := oneDay * 30
```

Comment:
[not blocking]
continuing what @cmwaters was saying, if we increase this even further, are there any consequences? like, if we just increase this to 6 monthts, can we get to the 1.5% mark?

Reply:
If we keep increasing it, the observed error in the inflation test will increase. I can try six months again but I expect the margin of error will exceed .01

Reply:
that's what I thought would happen, we don't have to worry about thanks for checking!!

Reply:
https://github.com/celestiaorg/celestia-app/pull/1791/commits/9c6d95ce7ed18810e04bb3c70631911241211473